src/NuGetPackageVerifier/CompositeRules/SigningVerificationCompositeRule.cs

```diff
@@ -15,6 +15,7 @@ public class SigningVerificationCompositeRule : IPackageVerifierRule
             new AuthenticodeSigningRule(),
             new PowerShellScriptIsSignedRule(),
             new RequiredNuSpecInfoRule(),
+            new DefaultCompositeRule(),
```

Comment:
There's a bunch of assemblies without `AssemblyDescription`. For some reason UniverseCoherence hasn't failed for this. Is this a newly introduced attribute?